app/web/src/AppRoute.tsx

```diff
@@ -58,86 +59,96 @@ export default function AppRoute({
-    <Route
-      {...otherProps}
-      render={({ location }) => (
-        <>
-          {isAuthenticated ? (
+  return (
+    <>
+      {isPrivate ? (
+        <Route
+          {...otherProps}
+          render={({ location }) => (
             <>
-              <>{variant !== "full-screen" && <Navigation />}</>
+              {isAuthenticated ? (
+                <>
+                  {variant !== "full-screen" && <Navigation />}
+                  <Container
+                    className={classNames({
+                      [classes.nonFullScreenStyles]: variant !== "full-screen",
+                      [classes.fullWidthContainer]: variant === "full-width",
+                      [classes.fullscreenContainer]: variant === "full-screen",
+                      [classes.standardContainer]: variant === "standard",
+                    })}
+                    maxWidth={
+                      variant === "full-screen" || variant === "full-width"
+                        ? false
+                        : "lg"
+                    }
+                  >
+                    {isJailed ? (
+                      <Redirect to={jailRoute} />
+                    ) : (
+                      <>
+                        <ErrorBoundary>{children}</ErrorBoundary>
+                      </>
+                    )}
+                  </Container>
+                </>
+              ) : (
+                <Redirect
+                  to={{
+                    pathname: loginRoute,
+                    state: { from: location },
+                  }}
+                />
+              )}
+            </>
+          )}
+        />
+      ) : (
+        <Route
+          {...otherProps}
+          render={() => (
+            <>
+              {variant !== "full-screen" && <Navigation />}
               <Container
                 className={classNames({
                   [classes.nonFullScreenStyles]: variant !== "full-screen",
-                  [classes.fullWidthContainer]: variant === "full-width",
                   [classes.fullscreenContainer]: variant === "full-screen",
+                  [classes.fullWidthContainer]: variant === "full-width",
                   [classes.standardContainer]: variant === "standard",
                 })}
                 maxWidth={
                   variant === "full-screen" || variant === "full-width"
                     ? false
-                    : undefined
+                    : "lg"
                 }
               >
-                {isJailed ? (
-                  <Redirect to={jailRoute} />
-                ) : (
-                  <>
-                    <ErrorBoundary>
-                      <Suspense
-                        fallback={
-                          <div className={classes.loader}>
-                            <CircularProgress />
-                          </div>
-                        }
-                      >
-                        {children}
-                      </Suspense>
```

Comment:
I think we still need this `Suspense` here since we are now lazy loading most routes. It's crashing on preview at the moment with this error in the console:

```
react-dom.production.min.js:216 Error: A React component suspended while rendering, but no fallback UI was specified.

Add a <Suspense fallback=...> component higher in the tree to provide a loading indicator or placeholder to display.
```